ffv1.md

```diff
@@ -23,17 +23,21 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 ## Definitions
 
 -------- --------------------------------------------------------------
+`Sample`: The smallest addressable representation of a primary color in a frame. Examples of sample are Luminance, Blue Chrominance, Red Chrominance, Alpha, Red, Green, Blue.
+
+`Pixel`: The smallest addressable representation of a color in a frame. It is composed of 1 or more samples.
```

Comment:
Worthwhile to mention subsamples here when pixels share samples?

Reply:
IMO useful for clear split between pixel and sample.
But no strong opinion on keeping it.

Reply:
Nit: strictly speaking, luma ≠ luminance and chroma ≠ chrominance.

Reply:
You mean you prefer me to write luma and chroma?

Reply:
No, I mean that, if I’ll find the time, I would like to go through the full document and distinguish between.